### DISCUSSION TITLE
v1.1.6

### DISCUSSION THREAD

**Discussion by jp7677** [avisi-cloud/structurizr-site-generatr#345] [Announcements]
## What's Changed
* Fix regression bug to get branch names. by @qtzar in https://github.com/avisi-cloud/structurizr-site-generatr/pull/343

**Full Changelog**: https://github.com/avisi-cloud/structurizr-site-generatr/compare/1.1.5...1.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/avisi-cloud/structurizr-site-generatr/releases/tag/1.1.6'>v1.1.6</a>.</em>

**Comment by dawidci-papaya** [comment]
Thank you for the new version. It's a really cool tool. I'm starting to use it at the company I work for.

@jp7677  Could you please update the Brew repository to the new version? https://github.com/avisi-cloud/homebrew-tools/pull/45

I would like to use it locally

**Reply by dirkgroot** [reply]
Thanks for pointing this out, totally forgot. I've updated the Brew repository.

**Reply by dawidci-papaya** [reply]
Thanks a lot !
